nas_ports/sysutils/tw_cli/Makefile

```diff
@@ -2,21 +2,22 @@
 
```

Comment:
Because I tried to build this branch, and I could not fetch this port anymore due the MASTER_SITE changed!